gui/library/Functions/Input.php

```diff
@@ -397,7 +397,7 @@ function getUserLoginDataForm($usernameRequired = true, $passwordRequired = true
                 ]
             ],
             'admin_pass_confirmation' => [
-                'password', ['validators' => [['Identical', true, ['admin_pass', 'messages' => tr('Passwords do not match.')]]]]
+                'password', ['validators' => [['Identical', true, ['admin_pass', 'messages' => tr('Passwords does not match.')]]]]
```

Comment:
"does" is wrong there. We are talking about the passwordS and not the password. This issue is at several place in your changes.